tests/visual/percy-storybook/Css.percy.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import DataTable from 'dash-table/dash/DataTable';
+
+const props = {
+    setProps: () => { },
+    data: [
+        { a: 1, b: 2, c: 3 },
+        { a: 2, b: 4, c: 6 },
+        { a: 3, b: 6, c: 9 }
+    ],
+    columns: [
+        { id: 'a', name: 'A' },
+        { id: 'b', name: 'B' },
+        { id: 'c', name: 'C' }
+    ],
+    css: [
+        { selector: 'td', rule: 'background-color: red !important;' }
+    ]
+};
+
+storiesOf('DashTable/CSS override', module)
+    .add('leading _ without letter', () => (<DataTable
+        {...props}
+        id={'_123'}
+    />))
+    .add('leading number', () => (<DataTable
+        {...props}
+        id={'123'}
+    />))
+    .add('escaped characters', () => (<DataTable
+        {...props}
+        id={'`~!@#$%^&*()=+ \\|/.,:;\'"`?[]<>{}'}
+    />))
+    .add('stringified object as id', () => (<DataTable
+        {...props}
+        id={{ id: 3, group: 'A' }}
```

Comment:
FWIW you're never going to actually get an object sent to the component by the renderer as an `id` - the renderer stringifies the id first, and does so as sorted JSON, so you'll get `'{"group":"A","id":3}'`

Does `DataTable` separately stringify ids? Because if I just call `CSS.escape` on an object it just gives `"\[object\ Object\]"`

Reply:
> Does DataTable separately stringify ids?

Oups.
https://github.com/plotly/dash-table/pull/766/commits/41981cbe86bb2006e7d14d3e363184f9a617154e